Client/mods/deathmatch/logic/CStaticFunctionDefinitions.cpp

```diff
@@ -6630,6 +6630,18 @@ bool CStaticFunctionDefinitions::SetHeatHaze(const SHeatHazeSettings& settings)
     return true;
 }
 
+bool CStaticFunctionDefinitions::ResetColorFilter()
+{
+    g_pMultiplayer->ResetColorFilter();
+    return true;
+}
+
+bool CStaticFunctionDefinitions::SetColorFilter(DWORD dwPass0Color, DWORD dwPass1Color)
+{
+    g_pMultiplayer->SetColorFilter(dwPass0Color, dwPass1Color);
+    return true;
+}
```

Comment:
don't need to add things to CStaticFuncDefs anymore, can just add content directly to CLuaWorldDefs

Reply:
@tederis Please fix this, and we can merge the PR.

Reply:
@saml1er Done